### ISSUE TITLE
when i train the dataset CUHK-SYSU,the result is not the same as the result you provide

### ISSUE THREAD

**Issue by Yx1322441675** [serend1p1ty/SeqNet#5]
My set is single GPU 2080ti ,i set the batchsize 1,its result is mAP=85.43%,top-1=86.72%.Do you know why?

**Comment by serend1p1ty** [comment]
You can get the same result as paper when you set batchsize=5, which requires about 28GB memory.
You can try batchsize=2, which will achieve better results than batchsize=1.
In the future, distributed training will be supported to solve the problem of the lack of GPU memory.

**Comment by serend1p1ty** [comment]
One more thing, batchsize=1 should match learning rate=0.0006.

**Comment by Yx1322441675** [comment]
Thank you very much

**Comment by Yx1322441675** [comment]
I have changed the lr=0.0006 when batchsize=1 ,its result get worse,
all detection:
  recall = 82.22%
  ap = 80.70%
search ranking:
  mAP = 85.20%
  top- 1 = 86.24%
  top- 5 = 94.17%
  top-10 = 95.45%
Maybe i should train more epoches?

**Comment by serend1p1ty** [comment]
BN is unstable when batchsize=1.
You can freeze BN or try a larger batchsize.

**Comment by ZhengPeng7** [comment]
The result is so great, I achieved almost the same result as the original paper showed (and even better) by training only two times:
| Best-Model | CUHK-SYSU(epoch-18) | PRW(epoch-17) |
| :--------: | :-----------------: | :-----------: |
|  vanilla   |    **94.13%-94.72%**    | **46.65%-83.91%** |
|    CBGM    |    94.80%-95.31%    | 47.42%-86.92% |

Thanks a lot for sharing this solid codebase.

**Comment by Wuzimeng** [comment]
I got similar result as @Yx1322441675 mentioned. Here I wonder whether you still remember how to achieve the same result as the original paper showed? @Yx1322441675 @ZhengPeng7

**Comment by ZhengPeng7** [comment]
He set batch size as 1. I didn't change any settings in the project. You may check that you kept the original batch size and image size, and use a close PyTorch version (I used 1.8.1, but I think all versions < 1.11 are okay) to use this project again.

**Comment by Wuzimeng** [comment]
Thanks, I used bs=4, lr=0.0024 without any other configs changed and my PyTorch version is 1.11. Maybe I should reduce it and have another try.

**Comment by ZhengPeng7** [comment]
You are welcome. I mentioned the PyTorch version since my friend met some nan problems using the too latest version (see [this issue](https://github.com/serend1p1ty/SeqNet/issues/26#issuecomment-1152114012)). And batch size also plays a role here to get exactly same results.

**Comment by serend1p1ty** [comment]
@Wuzimeng Could you share your train log?
